pd-configuration-file.md

```diff
@@ -253,6 +258,21 @@ PD 配置文件比命令行参数支持更多的选项。你可以在 [conf/conf
 + 同时进行的 Region Merge 调度的任务，设置为 0 则关闭 Region Merge。
 + 默认值：8
 
+### `witness-schedule-limit` <span class="version-mark">从 v6.6.0 版本开始引入</span>
+
++ 控制同时进行的 Witness 调度的任务个数。
```

Comment:
```suggestion
+ 控制并行 Witness 调度任务的个数。
```

Reply:
Number of concurrent witness scheduling tasks or Number of concurrent tasks that schedule witness replicas?